src/rust/ru.desu/res/source.json

```
@@ -4,7 +4,7 @@
 		"lang": "ru",
 		"name": "Desu",
 		"version": 2,
```

version should also be bumped so people can update
```suggestion
		"version": 3,
```